ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/search/presentation/FoundOccurrenceNode.java

```diff
@@ -140,6 +140,7 @@ public boolean isLeaf() {
   //TODO
   public void updatePresentation(@NotNull NodePresentation presentation) {
     SpanElement spanElement = createSpanElement(styles.styles().presentableTextContainer());
+    spanElement.setAttribute("file-name", itemPath);
```

Comment:
Is it possible use here element id instead of attribute it help [here](https://github.com/eclipse/che/pull/6395/files#diff-e6709237ba99803b449aad9102faa57aR414)

Reply:
Can we avoid this?

Reply:
Can you locate the element by text occurance?

Reply:
Name it `debugFilePath`